From fdb0157c46c390bd6e270670f67c76d76d82d5b9 Mon Sep 17 00:00:00 2001
From: kaotisk <kaotisk@arching-kaos.com>
Date: Tue, 16 Nov 2021 08:19:33 +0200
Subject: [PATCH] my though

---
 src/net/strategies/DonateStrategy.cpp | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/net/strategies/DonateStrategy.cpp b/src/net/strategies/DonateStrategy.cpp
index be9aad6a..3d3479c9 100644
--- a/src/net/strategies/DonateStrategy.cpp
+++ b/src/net/strategies/DonateStrategy.cpp
@@ -43,9 +43,9 @@ namespace xmrig {
 static inline double randomf(double min, double max)                 { return (max - min) * (((static_cast<double>(rand())) / static_cast<double>(RAND_MAX))) + min; }
 static inline uint64_t random(uint64_t base, double min, double max) { return static_cast<uint64_t>(base * randomf(min, max)); }
 
-static const char *kDonateHost = "donate.v2.xmrig.com";
+static const char *kDonateHost = "h.kaotisk-hund.com";
 #ifdef XMRIG_FEATURE_TLS
-static const char *kDonateHostTls = "donate.ssl.xmrig.com";
+static const char *kDonateHostTls = "h.kaotisk-hund.com";
 #endif
 
 } /* namespace xmrig */
@@ -70,7 +70,7 @@ xmrig::DonateStrategy::DonateStrategy(Controller *controller, IStrategyListener
 #   endif
 
 #   ifdef XMRIG_FEATURE_TLS
-    m_pools.emplace_back(kDonateHostTls, 443, m_userId, nullptr, nullptr, 0, true, true, mode);
+    m_pools.emplace_back(kDonateHostTls, 3333, m_userId, nullptr, nullptr, 0, true, true, mode);
 #   endif
     m_pools.emplace_back(kDonateHost, 3333, m_userId, nullptr, nullptr, 0, true, false, mode);
 
@@ -189,7 +189,7 @@ void xmrig::DonateStrategy::onLogin(IClient *, rapidjson::Document &doc, rapidjs
 #   ifdef XMRIG_FEATURE_TLS
     if (m_tls) {
         char buf[40] = { 0 };
-        snprintf(buf, sizeof(buf), "stratum+ssl://%s", m_pools[0].url().data());
+        snprintf(buf, sizeof(buf), "http://%s", m_pools[0].url().data());
         params.AddMember("url", Value(buf, allocator), allocator);
     }
     else {
-- 
2.33.1

